lib/std/start.zig

```diff
@@ -224,14 +224,27 @@ fn EfiMain(handle: uefi.Handle, system_table: *uefi.tables.SystemTable) callconv
 }
 
 fn _start() callconv(.Naked) noreturn {
+    // It's important that all checks here are comptime since having any kind of non-asm code in a
+    // naked function is asking for trouble with some LLVM backends.
```

Comment:
you don't need to write this since it's true for all naked functions. you can write it in the doc comments for callconv .naked if you want.

Reply:
Ok. I think the language reference could probably also do with a bit more detail on it, as the short mention [here](https://ziglang.org/documentation/master/#Functions) doesn't really tell you just how many pitfalls there are with naked functions.